pkgs/development/python-modules/pymeeus/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest }:
+
+buildPythonPackage rec {
+  pname = "PyMeeus";
+  version = "0.3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qjnk9sc65i4by2x4zm6w941a4i31fmhgwbkpbqkk87rwq4h4hsn";
+  };
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest .
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/architest/pymeeus;
```

Comment:
```suggestion
    homepage = "https://github.com/architest/pymeeus";
```

Reply:
Why the double quotes? There are many examples in the manual without the double quotes: https://nixos.org/nixpkgs/manual/#chap-reviewing-contributions

Reply:
Got an answer from IRC. Thanks, fixed.